packages/amplify-e2e-tests/package.json

```diff
@@ -17,7 +17,7 @@
   ],
   "private": true,
   "scripts": {
-    "e2e": "npm run setup-profile && jest --verbose",
+    "e2e": "npm run setup-profile && jest --verbose --forceExit",
```

Comment:
Why do we need this flag?

Reply:
One of the tests is succeeding but is not able to cleanup properly and because of that running background process, its not exiting the test. This flag does not effect the test outcome as such but just clears out left out processes. We will need to dig into this later.